packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -824,17 +824,23 @@ const tenantsRouter = (
         const ctx = fromAppContext(req.ctx);
 
         try {
-          validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE]);
+          validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE, M2M_ADMIN_ROLE]);
 
           const { tenantId, attributeId } = req.params;
-          await tenantService.revokeCertifiedAttributeById(
-            {
-              tenantId: unsafeBrandId(tenantId),
-              attributeId: unsafeBrandId(attributeId),
-            },
-            ctx
-          );
-          return res.status(204).send();
+          const { data: tenant, metadata } =
+            await tenantService.revokeCertifiedAttributeById(
+              {
+                tenantId: unsafeBrandId(tenantId),
+                attributeId: unsafeBrandId(attributeId),
+              },
+              ctx
+            );
+
+          setMetadataVersionHeader(res, metadata);
```

Comment:
Should be added to tenant API spec (see https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-a50f62f3a4939e0ad8d4f443bf76522090263ad0bfabf14e174d8d83475fc372R39)